packages/block-library/src/embed/wp-embed-preview.js

```diff
@@ -18,6 +20,36 @@ class WpEmbedPreview extends Component {
 		this.node = createRef();
 	}
 
+	componentDidMount() {
+		window.addEventListener( 'message', this.resizeWPembeds );
+	}
+
+	componentWillUnmount() {
+		window.removeEventListener( 'message', this.resizeWPembeds );
+	}
+
+	/**
+	 * Checks for WordPress embed event signaling the height
+	 * change when content loads.
+	 *
+	 * @param {WPSyntheticEvent} event Message event.
+	 */
+	resizeWPembeds( { data: { secret, message, value } = {} } ) {
```

Comment:
This all reads a bit like dark magic. :) I had never come across this height signalling before.

I'm not too confident to approve the change.

In any case, providing some context and linking to adequate prior art would be nice.

Reply:
You're right! I've updated the jsdoc.